erpnext/stock/get_item_details.py

```diff
@@ -209,7 +209,8 @@ def get_basic_details(args, item):
 			project: "",
 			qty: "",
 			stock_qty: "",
-			conversion_factor: ""
+			conversion_factor: "",
+			against_blanket_order: 1/0
```

Comment:
why `1/0`

Reply:
this is to be inline with the field type values for check box which is 1 or 0

Reply:
1/0 = ZeroDivisionError

Reply:
will change to be more clear

Reply:
Updated line. Copied line above `ignore_pricing_rule: 0/1`